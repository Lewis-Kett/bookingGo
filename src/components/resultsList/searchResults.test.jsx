import React from 'react';
import  SearchResults from './searchResults';
import renderer from 'react-test-renderer'


const data = [
    {name: 'test', placeType: 'test', city: 'test', country: 'test'},
    {name: 'test', placeType: 'test', city: 'test', country: 'test'},
    {name: 'test', placeType: 'test', city: 'test', country: 'test'}
]

describe('SearchResults', () => {

  it('should match snapshot', () => {

    const tree = renderer.create(<SearchResults searchResults={data} value={'test'} loaded={true} focused={true} />).toJSON()

    expect(tree).toMatchSnapshot()
  });

})
