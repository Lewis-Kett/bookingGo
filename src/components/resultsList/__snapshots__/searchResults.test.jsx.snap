// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults should match snapshot 1`] = `
<div
  className="listWrapper"
>
  <ul
    className="searchResultList"
  >
    <li
      className="searchResultItem"
    >
      <a
        className="searchResultLink"
        href="http://test.com"
      >
        <span
          className="searchResultPrefix "
        >
          
        </span>
        <div
          className="searchResultTitle"
        >
          <span
            className="searchResultName"
          >
            test
          </span>
          <span
            className="searchResultSubName"
          >
            test , test
          </span>
        </div>
      </a>
    </li>
    <li
      className="searchResultItem"
    >
      <a
        className="searchResultLink"
        href="http://test.com"
      >
        <span
          className="searchResultPrefix "
        >
          
        </span>
        <div
          className="searchResultTitle"
        >
          <span
            className="searchResultName"
          >
            test
          </span>
          <span
            className="searchResultSubName"
          >
            test , test
          </span>
        </div>
      </a>
    </li>
    <li
      className="searchResultItem"
    >
      <a
        className="searchResultLink"
        href="http://test.com"
      >
        <span
          className="searchResultPrefix "
        >
          
        </span>
        <div
          className="searchResultTitle"
        >
          <span
            className="searchResultName"
          >
            test
          </span>
          <span
            className="searchResultSubName"
          >
            test , test
          </span>
        </div>
      </a>
    </li>
  </ul>
</div>
`;
